vue/components/ui/radio-platforme/radio-platforme.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+    <div class="radio-group">
+        <label-platforme class="title" v-bind:text="labelTitle" v-if="labelTitle" />
+        <div class="radio-group-choices">
+            <div
+                class="radio-group-choice"
+                v-bind:class="{
+                    disabled: disabled || item.disabled,
+                    error: error || item.error,
+                    checked: item.value === value
+                }"
+                v-for="item in items"
+                v-bind:key="item.value"
+                v-on:click="onClick(item)"
+            >
+                <input type="radio" class="value" v-bind:id="item.value" />
+                <div class="circular-button" />
+                <label v-bind:for="item.value" class="label">
+                    {{ item.label ? item.label : item.value }}
+                </label>
+            </div>
+        </div>
+        <label-platforme
+            class="footer"
+            v-bind:size="'small'"
+            v-bind:text="labelFooter"
+            v-if="labelFooter"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.radio-group {
+    display: inline-block;
+}
+
+.radio-group > .title {
+    color: $pale-grey;
+    display: block;
+}
+
+.radio-group-choice {
+    cursor: pointer;
+    display: block;
+    line-height: 13px;
+    padding: 10px 6px 10px 6px;
+    user-select: none;
+}
+
+.radio-group-choice:hover {
+    background-color: #ebecf0;
+}
+
+.radio-group-choice:hover > .circular-button {
+    border-color: #c1c7d0;
+}
+
+.radio-group-choice:active {
+    background-color: #dde0e2;
+}
+
+.radio-group-choice.disabled > .label {
+    cursor: not-allowed;

```

Comment:
Is this needed? You already have the same thing on the parent

Reply:
Yes, because:
<img width="407" alt="imagem" src="https://user-images.githubusercontent.com/24736423/70053716-e86a9700-15cd-11ea-8e5c-37d6509344a2.png">